homeassistant/components/synology_dsm/__init__.py

```diff
@@ -64,7 +66,8 @@
     {
         vol.Required(CONF_HOST): cv.string,
         vol.Optional(CONF_PORT): cv.port,
-        vol.Optional(CONF_SSL, default=DEFAULT_SSL): cv.boolean,
+        vol.Optional(CONF_SSL, default=DEFAULT_USE_SSL): cv.boolean,
+        vol.Optional(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): cv.boolean,
```

Comment:
Even though this integration does support the configuration flow, we still do not accept changes to the YAML configuration.

This PR, therefore, violates ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

Reply:
@MartinHjelmare Should we revert?

Reply:
I think we should just remove the change to the config yaml schema. The rest is ok I think.

Reply:
Oh wait, we can drop yaml support 💁‍♂️

Reply:
Ok, I've seen that I'm on "Watch release" on the architecture repo, that explain things :sweat_smile: 

Do you think it's possible to make a release when a new ADR comes out ?
So it won't spam but let followers up to date.

Reply:
@Quentame Well... you have been notified of this 4 days ago as well: 

See: <https://github.com/home-assistant/core/pull/42539#discussion_r514198667>

Reply:
Indeed :disappointed: 
Was too quick on reading apparently.

To forget me, let me remove YAML support.

Reply:
Proposed #42777, in draft because of a question.